multimodal-search-pdf/pods/index-doc.yml

```diff
@@ -0,0 +1,32 @@
+jtype: BinaryPbIndexer # KV indexer to index docs key:doc.id, value:docs
+with:
+  index_filename: 'doc.gz'
+metas:
+  name: kv_indexer
+  workspace: $JINA_WORKSPACE/indexed
+requests:
+  use_default: true
+  on:
+    SearchRequest:
+      - jtype: ExcludeQL
+        with:
+          fields:
+            - buffer
+            - chunks
+          traversal_paths: [ 'c', 'cc' ]
+      - jtype: KVSearchDriver
+        with:
+          executor: kv_indexer
+    IndexRequest:
+      - jtype: ExcludeQL
+        with:
+          fields:
+            - chunks
+      #- jtype: URI2DataURI {} # IMPORTANT: URI2DataURI can convert local uri to data uri, uncomment this to get URI data in resp
```

Comment:
clean up
```suggestion
```

Reply:
Maybe we just keep it as a comment? I think this is important because if users want to have dataURI, then they can use this.